packages/block-library/src/buttons/deprecated.js

```diff
@@ -29,6 +29,8 @@ const migrateWithLayout = ( attributes ) => {
 				orientation: orientation || 'horizontal',
 			},
 		} );
+		delete updatedAttributes.contentJustification;
```

Comment:
I think we can simplify in the destructuring above.

`const { contentJustification, orientation, ...updatedAttributes } = attributes;

Also we might want only to add layout's `justifyContent` or `orientation` props only if the respective attribute exists so as to get the defaults from `flex.layout`.

Reply:
Fixed!